packages/website/src/content/docs/sidecar/vite.mdx

```diff
@@ -14,12 +14,12 @@ We provide a Vite plugin that automatically starts & stops Sidecar for you when
   </TabItem>
   <TabItem label="pnpm">
     ```sh
-    pnpm add @spotlightjs/spotlight --save-dev
```

Comment:
Hi @alesvaupotic this command is correct. 
Other places are affected I guess.

it should @spotlightjs/spotlight

cc @Lms24

Reply:
Hi, @alesvaupotic and @Shubhdeep12 good catches, thanks! I agree, this is a bug in the `spotlightjs/spotlight` package. it should also export the vite plugin. Will fix this asap.

Reply:
Thanks, I was wondering what I was doing wrong. Trying to work it out for SvelteKit app ...

Reply:
TLDR: I'd recommend setting up the sidecar within the server hooks. Easy to do and you don't need to mess around with vite plugins or npx commands.